R/components.R

```diff
@@ -374,9 +374,23 @@ OverflowList <- component("OverflowList")
 #'
 #' @example inst/examples/PanelStack2.R
 #' @inherit template params
+#' @param panels List of lists - each list contains `title` (string) and `content` (HTML)
+#' @param ns Namespace of given panel stack (required if there's more than 1 panel stack)
+#' @param size Numeric vector of length 2 - `c(width, height)`
 #' @export
 PanelStack2 <- component("PanelStack2")
 
+#' @rdname PanelStack2
+#' @export
+PanelStack <- function(panels, ns = "ps", size = c(300, 250), ...) {
```

Comment:
Naming: Let's use `PanelStack` for the raw React component and `PanelStack.shinyWrapper` for our wrapper component.

Reply:
Renamed